src/main/java/stream/flarebot/flarebot/util/pagination/PagedEmbedBuilder.java

```diff
@@ -65,13 +68,39 @@ public PagedEmbed build() {
         boolean pageCounts = false;
         if (list.getPages() > 1)
             pageCounts = true;
-        return new PagedEmbed(title, codeBlock, hasCodeBlock, list, pageCounts, color);
+        return new PagedEmbed(title, codeBlock, hasCodeBlock, list, pageCounts, color, groupPrefix, groupsPerPage);
     }
 
     public void setColor(Color color) {
         this.color = color;
     }
 
+    /**
+     * Enables the embed to use the groups within the {@link PaginationList}.
+     */
+    public void useGroups() {
+        setGroupPrefix("");
```

Comment:
Also stop setting these, and for the 100th time, use the root useGroups with the default global variable

Reply:
if i don't set them I can't tell if it's enabled

Reply:
also why whould i run the method with the default if the default is allready set